docs/guide/setting-up.md

```diff
@@ -18,12 +17,10 @@ node -v && npm -v
 # if installed.
 mongod --version
 # This will display your MongoDB version, if installed.
-which convert && which identify
-# This will display the location of the ImageMagick utilities, if installed.
 ```
 
 ::: tip NOTE
-ImageMagick is optional, but recommended. It provides the `convert` and `identify` command line tools, which Apostrophe uses to scale and crop images quickly. If you do not install it Apostrophe can still handle image uploads, though more slowly.
+[ImageMagick](https://imagemagick.org/script/download.php) can be used if your system doesn't support the default npm [`sharp` package](https://www.npmjs.com/package/sharp). It provides the `convert` and `identify` command line tools, which Apostrophe uses to scale and crop images quickly. If your project does not contain `sharp` and ImageMagick is installed, apostrophe will fall-back automatically.
```

Comment:
"If your project does not contain `sharp`" is confusing, it makes it sound like you have to install sharp.

"Apostrophe will automatically install `sharp` when you `npm install` your project, you do not have to add it yourself. If for any reason Apostrophe is unable to install its `sharp` dependency and `ImageMagick` has been installed on your system, Apostrophe will fall back to `ImageMagick` automatically."

Reply:
Changed.